packages/gitmoji-changelog-cli/src/index.js

```diff
@@ -1,9 +1,12 @@
 #! /usr/bin/env node
-import fs from 'fs'
-import { changelog } from '@gitmoji-changelog/core'
-import { convert } from '@gitmoji-changelog/markdown'
 
-export async function main() {
+'use strict'
```

Comment:
Why is the purpose here ? In node + a linter to help you wrote a better code ? 
(Serious question, I ask myself if I should put that on my node scripts)

Reply:
I wonder if running node with `--check` flag and `'use strict'` make any difference to the syntax analysis.

Reply:
Ok, then I will not use it on my own code :) 

 - linter is better
 - ci should check the code (with linter) before running a deployment

Thank you guys